apollo.config.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config({ path: '.env.development.local' });
```

Comment:
We should use a pattern in here ? for example `.env.*` ? Or you only need apollo on local?

Reply:
I think it's only needed locally unless we want to generate TS types for queries whenever we invoke Travis for example.
What do you think?

Reply:
I dont have any plan for it so i guess if you think local is fine then its fine